### ISSUE TITLE
Blender.org: Blender Conference content is hidden on the main page

### ISSUE THREAD

**Issue by ggrzybow** [endolith/clickbait#10]
How to reproduce:

1. Go to https://www.blender.org/
2. Top section (under the white menu bar on top) has a gray backround. On the right, there are colorful flowers. On the left, there is nothing.
3. Disable cosmetic filtering and see that on the left, content appears:

>  Blender Conference
> Thanks everyone for an amazing time!
> (two buttons here)

4. This content should not be hidden in the first place.

Looks like it's hidden because it has a `featured-content` class.

**Comment by endolith** [comment]
I agree, but am busy with holidays, if you want to figure out the right change to the filters and submit a pull request. otherwise I'll get to it eventually

**Comment by ggrzybow** [comment]
I'm not in a hurry, don't worry, take your time. :)

I'm not even sure what would constitute a proper fix. Allowing `featured-content` on blender.org? Blocking `featured-content` only on certain websites? Something else?
